Prevents playing multiple cards at once

j-mao opened the pull request:
Also has simplified implementation of anti-discard.
Did I even need a new pull request?  Who knows

j-mao committed:
Stops discarding cards when not discarding kitty

Alternate implementation

j-mao committed:
Stops discarding cards when not discarding kitty

Only need one button, instead of both play and discard..  server figures out what to do

DXsmiley commented:
This design seems better than the last one. However, I think that the buttons should be labelled with 'play' or 'discard' depending on what they will actually do.

I also think that the other players should be prevented from playing during the discard phase (either that, or let the discarding player play on top of their three discarded cards).

You don't need to make a new pull request (I think), although I don't know how to modify an existing one.

Also, if you could use the values `True` and `False` in python, that would be appreciated (e.g. server.py, line 92).

dxsmiley commented on server.py:
```diff
@@ -83,15 +83,31 @@ def page_action():
+				if len(game_data['hands'][player]) <= 10:
+					# in this case, play a card
+					# make sure that only one card is played per turn
+					already_played = 0
```
`True` and `False` is preferred.

(I know I said this elsewhere, I just wanted an excuse to try out this feature :smile:)

j-mao committed:
Changed ints to bools

Edited on phone and untested, no guarantees that it will work.  It's probably fine anyway.

j-mao commented:
Might be fixed now.
